cfme/tests/webui/test_advanced_search.py

```diff
@@ -297,7 +296,6 @@ class TestContainers(object):
 
 
 @inject_tests
-@pytest.mark.meta(blockers=[GH('ManageIQ/integration_tests:9723')])
 class TestAnsibleTower(object):
     params_values = [
         SearchParam('ansible_tower_providers', 'All', 'ansible_tower_explorer_provider',
```

Comment:
There is no longer any `ansible_tower_providers` collection. I think these tests should be merged into the `TestConfigManagement` class if possible.